Sources/Register.php

```diff
@@ -429,6 +448,9 @@ function Register2()
 		$_POST['options'] = isset($_POST['options']) ? $_POST['options'] + $_POST['default_options'] : $_POST['default_options'];
 	$regOptions['theme_vars'] = isset($_POST['options']) && is_array($_POST['options']) ? $_POST['options'] : array();
 
+	// Note when they accepted the agreement and privacy policy
+	$regOptions['theme_vars']['agreement_accepted'] = $regOptions['theme_vars']['policy_accepted'] = time();
```

Comment:
This should be changed to:
```suggestion
	if (!empty($modSettings['requireAgreement']))
		$regOptions['theme_vars']['agreement_accepted'] = time();
	if (!empty($modSettings['requirePolicyAgreement']))
		$regOptions['theme_vars']['policy_accepted'] = time();
```
(Note to self and other dev team members: We should probably change that in 2.0.x, too.)